packages/connect-node/src/node-universal-client.ts

```diff
@@ -553,8 +551,10 @@ function createSentinel(signal?: AbortSignal): Sentinel {
   const s = Object.assign(p, c);
 
   function onSignalAbort() {
+    // AbortSignal.reason was added in Node.js 17.2.0, we cannot rely on it.
     c.reject(
-      new ConnectError("operation was aborted via signal", Code.Canceled)
+      signal?.reason ??
+        new ConnectError("This operation was aborted", Code.Canceled)
```

Comment:
Unrelated to the bug fixes. With Node.js v16 being EOL soon, we can make better use of AbortSignal.reason.